conf/messages.en

```diff
@@ -161,5 +161,6 @@ document.removeDocument.error.required = Select yes if you want to remove the {0
 
 document.attachToAllItems.title = Do you want to use this document for all items?
 document.attachToAllItems.heading = Do you want to use this document for all items?
-document.attachToAllItems.checkYourAnswersLabel = Attach To All Items
+document.attachToAllItems.checkYourAnswersLabel = Do you want to use this document for all items?
+document.attachToAllItems.change.hidden = if you want to use this document for all items
```

Comment:
Is this intended to be a lower case 'if'?

Reply:
Guessing it is looking at the tests :-)

Reply:
Yep it gets prepended with "Change ..." for the visually hidden text :) Thanks for reviewing!